features/page_objects/deregistration_page.rb

```diff
@@ -15,4 +15,12 @@ def revoke(args = {})
     deregister_button.click
   end
 
+  def cease(args = {})
```

Comment:
I'd recommend sticking with the pattern of calling the method which fills in and submits a page `submit`, and pass in an arg which determines whether you want to cease or revoke the registration.